### PR TITLE
checkBox.js: remove extra unused argument which was causing a warning…

### PR DESCRIPTION
**Pull request by collinss**
… with new cjs/mozjs

### CODE REVIEW

File: js/ui/checkBox.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ CheckBoxContainer.prototype = {
         let min = minBoxWidth + minLabelWidth + this._spacing;
         let nat = natBoxWidth + natLabelWidth + this._spacing;
         [min, nat] = node.adjust_preferred_width(min, nat);
```
Comment (In-line) [inline js/ui/checkBox.js]: This is unnecessary change

File: js/ui/checkBox.js [hunk 2]
```diff
@@ -52,7 +52,7 @@ CheckBoxContainer.prototype = {
...
-        
+
```
Comment (In-line) [inline js/ui/checkBox.js]: This is unnecessary change.

Comment (leigh123linux) [inline js/ui/checkBox.js]: I don't see any issue removing white-space.

Comment (JosephMcc) [inline js/ui/checkBox.js]: I don't see the issue here either.

Comment (mtwebster) [inline js/ui/checkBox.js]: There's no issue with removing nearby line that is just whitespace.. we try to leave code cleaner than it was - so picking up the trash around the area you're working is ok.  It's the needless reformatting of swaths of code for no other reason than appearance that is frowned upon.
